src/main/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/service/PartATemplateReplacementService.kt

```diff
@@ -77,7 +78,14 @@ internal class PartATemplateReplacementService {
       "fax_number" to partAData.localPoliceContact?.faxNumber,
       "email_address" to partAData.localPoliceContact?.emailAddress,
       "has_vulnerabilities" to if (hasVulnerabilities(partAData)) YES else NO,
-      "gender" to partAData.gender
+      "gender" to partAData.gender,
+      "date_of_birth" to partAData.dateOfBirth?.format(DateTimeFormatter.ofPattern("dd/MM/yyyy")),
+      "name" to partAData.name,
+      "ethnicity" to if (partAData.ethnicity.isNullOrBlank()) "Not specified" else partAData.ethnicity,
```

Comment:
minor point but is it worth moving the `Not specified` a variable? Reckon this could come up a few times.

Reply:
Neat point, I added an entry to MrdConstants